packages/twenty-server/src/workspace/cron/clean-inactive-workspaces/clean-inactive-workspace.job.ts

```diff
@@ -247,6 +247,8 @@ export class CleanInactiveWorkspaceJob
   async handle(data: CleanInactiveWorkspacesCommandOptions): Promise<void> {
     const isDryRun = data.dryRun || false;
 
+    this.workspacesToDelete = [];
```

Comment:
Then I'm wondering if we don't want to simply init the array here and not store it on the class level, imho it will avoid this kind of issue and I don't see a strong benefit here (we will have to pass the array for some functions but I think it's fine?).
Also I think deleteWorkspace method name is confusing because it's actually not deleting the workspace it seems?

Reply:
Yes, I can indeed use a local variable. The naming is not up to date indeed, I can change it to match the new behavior.

Reply:
Thanks! Otherwise lgtm